stable/stable_rec/network-stresser/templates/stresser-receiver-rc.yml

```diff
@@ -42,8 +42,8 @@ spec:
         - --use_iperf={{ .Values.cmdOptions.useIperf }}
         - --tcp_receiver={{ .Values.imageConfig.tcpReceiverPath }}
         - --udp_receiver={{ .Values.imageConfig.udpReceiverPath }}
-        - --tcp_port={{ .Values.cmdOptions.tcpPort }}
-        - --udp_port={{ .Values.cmdOptions.udpPort }}
+        - --tcp_port=8081
```

Comment:
Why you moved to fix values, can you change the defaults outside and not change to fix numbers ?